packages/sdk/test/evm/sdk-init.ts

```diff
@@ -0,0 +1,42 @@
+import { ThirdwebSDK } from "../../src/evm";
+import { Goerli, Fncy } from "@thirdweb-dev/chains";
+// import { signers, storage } from "./before-setup";
+// import { AddressZero } from "@ethersproject/constants";
+// import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+
+global.fetch = require("cross-fetch");
+
+describe("SDK Initialization", async () => {
+  // let adminWallet: SignerWithAddress, samWallet: SignerWithAddress;
```

Comment:
Cleanup

Reply:
was gonna also test it with signer etc but yeah